scalars/contributed/jakobmerrild/long.md

```diff
@@ -0,0 +1,79 @@
+<!-- cspell:ignore <github user name> -->
+
+# Long — GraphQL Custom Scalar
+
+Author - jakobmerrild
+
+Date - 2024-12-16
+
+**License and Copyright**
+
+Copyright © GraphQL contributors. This specification is licensed under
+[OWFa 1.0](https://www.openwebfoundation.org/the-agreements/the-owf-1-0-agreements-granted-claims/owfa-1-0).
+
+# Overview
+
+This Scalar represents a 64-bit signed integer (non-fractional) value, ranging from `-2^63` to `2^63-1`. Such values are commonly
+used to represent timestamps as the number of milliseconds since the UNIX epoch (`1970-01-01T00:00:00.000Z`)
+among other use cases.
```

Comment:
I would remove the reference to timestamp, I don't think it's specifically relevant here and makes the spec longer to process.

```suggestion
This scalar represents a 64-bit signed integer (non-fractional) value, ranging from `-2^63` to `2^63-1`. ```